test/include/nntrainer_test_util.h

```diff
@@ -212,6 +213,20 @@ const std::string config_str2 = "[Network]"
     }                                                        \
   } while (0)
 
+
+/**
+ * @brief test tensor is equal
+ * @param[in] Tensor A, tensor to be compared
+ * @param[in] Tensor B, tensor to be compared
+ */
+void test_tensor_eq(nntrainer::Tensor const &A, nntrainer::Tensor const &B);
+
+/**
+ * @brief return a tensor filled with contant value with dimension
+ */
+nntrainer::Tensor constant(float value, unsigned int batch, unsigned channel,
```

Comment:
Similarly this can also be added into `tensor class` if you think it might be useful in the library itself.